.github/workflows/cd.yml

```
@@ -0,0 +1,48 @@
+name: bosdyn_msgs CD
+
+on:
+  push:
+    tags:
+      - '*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release-bundles-amd64:
+    runs-on:
+      group: bdai-gh-large-runners
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Build bosdyn_msgs bundles
+      run: make -j8 VERSION=${{ github.ref_name }} amd64
```

@amessing-bdai hmm, in hindsight, I could have allowed `debhelper` to go crazy on those cores:
```suggestion
      run: make -j VERSION=${{ github.ref_name }} amd64
```
We risk `gcc` exhausting runners' memory, but it will build faster if it doesn't.